README.md

```diff
@@ -24,6 +24,18 @@ custom:
     distributionFolder: client/dist # (Optional) The location of your website. This defaults to client/dist
     indexDocument: index.html # (Optional) The name of your index document inside your distributionFolder. Defaults to index.html
     errorDocument: error.html # (Optional) The name of your error document inside your distributionFolder. Defaults to error.html
+    objectMetadata: # (Optional) S3 object metadata (HTTP headers) for hosted files
```

Comment:
Does this PR support metadata other than headers? If not, maybe this option should be changed to a more specific name, like `objectHeaders`

Reply:
As far as I can tell, 'metadata' is just AWS's term for HTTP headers on objects, so I used the terminology to be consistent with their naming. I'd be fine renaming it to `objectHeaders` though